src/WebJobs.Script/Utility.cs

```diff
@@ -924,28 +924,32 @@ public static bool IsMediaTypeOctetOrMultipart(MediaTypeHeaderValue mediaType)
             }
         }
 
-        public static bool CanWorkerIndex(IEnumerable<RpcWorkerConfig> workerConfigs, IEnvironment environment)
+        public static bool CanWorkerIndex(IEnumerable<RpcWorkerConfig> workerConfigs, IEnvironment environment, FunctionsHostingConfigOptions functionsHostingConfigOptions)
         {
-            if (!FeatureFlags.IsEnabled(ScriptConstants.FeatureFlagEnableWorkerIndexing, environment))
+            string appName = environment.GetAzureWebsiteUniqueSlotName();
```

Comment:
does this also need to have `ToLowerInvariant()` called?

Reply:
`ToLowerInvariant()` is already being called on it.
https://github.com/Azure/azure-functions-host/blob/7fd7c77947535a6b68d2b4bf5416fc8302bd31d9/src/WebJobs.Script/Environment/EnvironmentExtensions.cs#L355-L367